hack/rpm-local-build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+
+if [[ ! -d dist/rpmbuild  ]]; then
+	echo "Cannot build as artifacts are not generated. Run scrips/rpm-prepare.sh first"
+	exit 1
```

Comment:
any reason not to just run it?

Reply:
This script is taken from [openshift/odo](https://github.com/openshift/odo/blob/main/scripts/rpm-local-build.sh). Right now I'm keeping the script as similar to their as possible. If this  can be optimized I will do that after we have generated our first release.